aten/src/ATen/native/cuda/Reduce.cuh

```diff
@@ -25,11 +25,22 @@ static inline int64_t div_up(int64_t a, int64_t b) {
   return (a + b - 1) / b;
 }
 
+// returns round down to the closest integer of log2(n)
```

Comment:
This sentence is a bit difficult to understand. I think it means "[floor](https://en.wikipedia.org/wiki/Floor_and_ceiling_functions) of `log2(n)`" -- is that right?

Reply:
Yep that sounds right.
let me push an update for this

Reply:
And apologize for my previous confusing & incorrect comment.